src/main/src/main/java/org/geoserver/catalog/ResourcePool.java

```diff
@@ -1,4 +1,4 @@
-/* (c) 2014 - 2017 Open Source Geospatial Foundation - all rights reserved
+/* (c) 2014 - 2020 Open Source Geospatial Foundation - all rights reserved
```

Comment:
No requirement to update the dates anymore, we have determined it has no legal value.

Reply:
Should I revert the dates?

Reply:
@jodygarnett do we have to revert the dates?

Reply:
No value to update the dates (no harm in it either) --> no need to revert.

aside: We do ask that a date be added when a new file is created.